PCbuild/pythoncore.vcxproj

```
@@ -415,6 +415,7 @@
     <ClCompile Include="..\Modules\sha1module.c" />
     <ClCompile Include="..\Modules\_hacl\Hacl_Streaming_SHA2.c" />
     <ClCompile Include="..\Modules\sha2module.c" />
+    <ClCompile Include="..\Modules\_hacl\Hacl_Hash_SHA3.c" />
```

Should this be moved to keep the list alphabetical? (pt. 2)
```suggestion
    <ClCompile Include="..\Modules\_hacl\Hacl_Hash_SHA3.c" />
    <ClCompile Include="..\Modules\sha3module.c" />
```